### PR TITLE
Artemis and Artemis_idfm as requirements

### PR DESCRIPTION
**Pull request by nonifier**
* Add script that checks whether a job on Jenkins is OK or not.
* Travis should now fail is Artemis or Artemis_idfm is broken.

You even have colours and emojis for free ! :rainbow: :unicorn: :rainbow: 

Example of failing job : https://travis-ci.com/nonifier/navitia/jobs/242037248

### CODE REVIEW

File: source/scripts/checkJenkinsJob.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    echo " - job: a string that defines the job's name to be checked"
+    echo ""
+    exit 1
+fi
+
+if [[ -z $JOB_COLOUR ]]; then
+    echo -e "${RED}Jenkins job '$JENKINS_JOB' does not exist${SET}" 1>&2
+    exit 2
+elif [[ "$JOB_COLOUR" != '"blue"' ]]; then
+    echo -e "${RED} 🔥 Jenkins job '$JENKINS_JOB' is broken 🔥, please cease all acticity and go fix it 🚒 " 1>&2
```
Comment (pbougue) [inline source/scripts/checkJenkinsJob.sh:25]:
```suggestion
    echo -e "${RED} 🔥 Jenkins job '$JENKINS_JOB' is broken 🔥, please cease all activity and go fix it 🚒 " 1>&2
```

**Review by kinnou02 — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -14,6 +14,8 @@ addons:
     packages:
     - clang-format-6.0
     - rabbitmq-server
+    - jq
```
Comment (benoit-bst) [inline .travis.yml:17]: `jq` is available on dist xenial ? Nice

File: source/scripts/checkJenkinsJob.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    echo ""
+    echo " - job: a string that defines the job's name to be checked"
+    echo ""
+    exit 1
+fi
+
+if [[ -z $JOB_COLOUR ]]; then
+    echo -e "${RED}Jenkins job '$JENKINS_JOB' does not exist${SET}" 1>&2
+    exit 2
+elif [[ "$JOB_COLOUR" != '"blue"' ]]; then
```
Comment (benoit-bst) [inline source/scripts/checkJenkinsJob.sh:24]: Blue powa !!

**Review by benoit-bst — APPROVED** [review]
I guess you have tested that.

**Review by pbougue — APPROVED** [review]
I don't know how to feel about this :grin: 

This PR is very nice, useful and shinny :sparkles: 
But at the same time I have now seen that api and the "color" status :nauseated_face:  
Gonna be hard for private jobs and if we migrate, but we'll have time to migrate script at some point (maybe to jenkins itself?).

**Review by mbounabi — APPROVED** [review]

**Pull request merged**